packages/esify/index.js

```diff
@@ -20,6 +20,16 @@ var TRANSFORMS = [
   {path: 'shopify-codemod/transforms/function-to-arrow'},
   {path: 'shopify-codemod/transforms/global-assignment-to-default-export', test: false},
   {path: 'shopify-codemod/transforms/global-reference-to-import'},
+  {path: 'shopify-codemod/transforms/assert/assert-to-assert-ok', test: true},
+  {path: 'shopify-codemod/transforms/assert/negated-assert-ok-to-assert-not-ok', test: true},
```

Comment:
Indeed.   Also `move-literals-to-expected-argument' and `equality-comparisons-to-assertions`.

I was going to be lazy and just document the dependency in the README, but I'm open to doing something more bullet proof.